Fix protool docker image to stop downloading tools on every run.

```docker run --rm -v /Users/romanuseinov/go/src/github.com/iov-one/weave:/work iov1/prototool:v0.2.0 prototool format -w
error downloading https://github.com/protocolbuffers/protobuf/releases/download/v3.7.1/protoc-3.7.1-linux-x86_64.zip: Get https://github.com/protocolbuffers/protobuf/releases/download/v3.7.1/protoc-3.7.1-linux-x86_64.zip: EOF
Make sure GitHub Releases has a proper protoc zip file of the form protoc-VERSION-OS-ARCH.zip at https://github.com/protocolbuffers/protobuf/releases/v3.7.1
Note that many micro versions do not have this, and no version before 3.0.0-beta-2 has this
make: *** [protofmt] Error 1```

I'm pretty sure this can be done by simply running all the needed tools as part of the Dockerfile initialisation to cache them.

--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@ MODE ?= set
 
 # for dockerized prototool
 USER := $(shell id -u):$(shell id -g)
-DOCKER_BASE := docker run --rm -v $(shell pwd):/work iov1/prototool:v0.2.0
+DOCKER_BASE := docker run --rm -v $(shell pwd):/work iov1/prototool:0.2.1
 PROTOTOOL := $(DOCKER_BASE) prototool
 PROTOC := $(DOCKER_BASE) protoc
 

--- a/scripts/build_protodocs_docker.sh
+++ b/scripts/build_protodocs_docker.sh
@@ -5,8 +5,8 @@ set -o errexit -o nounset -o pipefail
 # This is a version of build_protodocs, which uses docker images to build, and includes gogoproto support,
 # so it doesn't need the pre-clean step
 
-protoc="docker run --rm -v $(pwd):/work iov1/prototool:v0.2.0 protoc"
-prototool="docker run --rm -v $(pwd):/work iov1/prototool:v0.2.0 prototool"
+protoc="docker run --rm -v $(pwd):/work iov1/prototool:v0.2.1 protoc"
+prototool="docker run --rm -v $(pwd):/work iov1/prototool:v0.2.1 prototool"
 
 files=$(${prototool} files | grep -v examples | grep -v cmd/bcpd | sort)
 ${protoc} -I . -I /usr/include --doc_out=docs/proto --doc_opt=html,index.html ${files}

